### DISCUSSION TITLE
Login inicial no Homer

### DISCUSSION THREAD

**Discussion by vyctorvalmorbida** [sipcapture/homer#631] [Q&A]
Olá pessoal, não estou conseguindo realizar o primeiro login na plataforma, digito o login "admin" e senha "sipcapture" mas quando clico em "Login" não acontece nada. Alguém sabe me dizer o que devo fazer?

![Captura de Tela 2024-03-05 às 11 01 13](https://github.com/sipcapture/homer/assets/83193192/ae389aaa-1362-49b0-a87c-be1f51cb9052)

**Comment by Dletta** [comment]
Can you confirm that the database is online and reachable by homer?
